invenio_rdm_records/vocabularies/resource_types.csv

```diff
@@ -0,0 +1,3 @@
+type,type_name,subtype,subtype_name,datacite_general,datacite_type,csl,openaire_resourceType,openaire_type,schema.org,notes
```

Comment:
Is `datacite_type` necessary? i.e. can it always be filled with the `subtype_name`?

Reply:
DataCite doesn't have a controlled vocabulary for subtype, so in terms of validating the metadata it would be fine. Datacite does internally use subtypes for some metadata formatting (e.g. using "Dissertation" will result in BibTex with the correct item type). So having the mapping could be useful if you wanted to have different labels in the repository such as "Thesis" and map into "Dissertation" for DataCite.

Reply:
Good to know! Will keep those for now then.